onboard/catkin_ws/src/cv/scripts/detection.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+import rospy
+import os
+import yaml
+from custom_msgs.msg import Object
+from custom_msgs.srv import ToggleModel
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge
+from detecto.core import Model
+
+
+class Detector:
+
+    # Load in models and other misc. setup work
+    def __init__(self):
+        rospy.init_node('cv', anonymous=True)
+
+        self.bridge = CvBridge()
+        self.camera = rospy.get_param('~/{}/camera'.format(rospy.get_name()))
```

Comment:
I think that this can be just `rospy.get_param("~camera")`, but can't be sure without testing.

Reply:
Yep that works